src/reducers.js

```diff
@@ -87,6 +89,68 @@ export function Reducer(prevState = {}, action) {
   if (!newState.pathQueries)
     newState.pathQueries = [];
 
+  // update url after state change unless on redux initialization or
+  // explicitly bypassed
+  if (action.updateUrl === true && prevState !== undefined)
+    updateUrl(newState);
```

Comment:
In the past, it has usually been easier to avoid modifying the url after a state change and only do the other way around: url change --> triggers state change. Also, if this has any side effects it might not be good to include it in the reducer, maybe in a middleware or action creator?

Reply:
When you've done that in the past, was it in the case where all of the global state was able to be stored in the url? Because in our case that's not possible because there's too much info.

I'm still trying to think/experiment with how it would look like in my case to do this approach, where the flow is from url -> state for some variables, but not others.

Reply:
> When you've done that in the past, was it in the case where all of the global state was able to be stored in the url?

No, some variables were in the state and others depended of ajax queries to an API.

Reply:
I created #65 to look into this at a later time, since I think I have a working and somewhat intuitive (at least to me) solution in this PR.